randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -902,6 +904,8 @@ def get_random_size(minimum, maximum):
             "hasSpoiler": self.description.has_spoiler,
             "roomRandoMode": self.configuration.room_rando.value,
 
+            "randEnemyAttributes": self.configuration.enemy_attributes and self.configuration.enemy_attributes.as_json
```

Comment:
`self.configuration.enemy_attributes.as_json if self.configuration.enemy_attributes is not None else None,`